ckanext/datastore/interfaces.py

```diff
@@ -232,3 +235,82 @@ def update_datastore_info_field(
         in the data dictionary page.
         """
         return field
+
+
+class IDatastoreDump(interfaces.Interface):
```

Comment:
Interfaces shouldn't have any code in them, this could be a normal base class (e.g. `DatastoreDumpFormat`) instead